mock/requirements.txt

```diff
@@ -4,3 +4,4 @@ pyroute2
 requests
 rpmautospec-core
 templated-dictionary
+backoff
```

Comment:
Thank you, would you mind moving it up to keep the alphabetical order?

Reply:
Sure.

Reply:
Thank you!